channels/al.m3u

```diff
@@ -13,8 +13,8 @@ https://shkoder.gjirafa.com/api/media/rgjirafa/t0110y/index.m3u8
 http://93.157.62.180/Cufo/index.m3u8
 #EXTINF:-1 tvg-id="FrameTV.al" tvg-country="AL" tvg-language="" tvg-logo="" group-title="",Frame TV (720p) [Not 24/7]
 http://195.154.252.221:8000/play/a002/index.m3u8
-#EXTINF:-1 tvg-id="KLAN.al" tvg-country="AL" tvg-language="" tvg-logo="https://i.imgur.com/pWoMc0r.png" group-title="",KLAN [Offline]
-http://79.106.73.244:4040/live/klanhdmob/playlist.m3u8
+#EXTINF:-1 tvg-id="KLAN.al" tvg-country="AL" tvg-language="" tvg-logo="https://i.imgur.com/pWoMc0r.png" group-title="",KLAN (720p)
+http://93.157.62.180/KlanPlus/index.m3u8
```

Comment:
This stream is Klan Plus, not Klan.

Reply:
But content of channel belongs Klan

Reply:
Klan Plus and Klan are two distinct channels, if you look at the playlist, which is why i'm saying it.

Reply:
ok content of channel is klan plus